tailwind.config.js

```diff
@@ -0,0 +1,32 @@
+import {readFileSync} from 'node:fs';
+import {parse} from 'css-variables-parser';
+
+const colors = Object.keys(parse([
+  readFileSync(new URL('web_src/css/themes/theme-gitea-light.css', import.meta.url), 'utf8'),
+  readFileSync(new URL('web_src/css/themes/theme-gitea-dark.css', import.meta.url), 'utf8'),
+].join('\n'), {})).filter((prop) => prop.startsWith('color-')).map((prop) => prop.substring(6));
+
+/** @type {import('tailwindcss').Config} */
+export default {
+  prefix: 'tw-',
+  content: [
+    './templates/**/*.tmpl',
```

Comment:
Does the tailwind processor work well with "tmpl" files?

Could there be some "devtest" examples to show how it works?

Reply:
Yes it seems to work fine as long as the strings are unbroken. Don't want to bother with examples now, usage should be obvious anyways.

Reply:
Added a sample:

<img width="171" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/c8c46368-4b98-4f13-a11d-862c34bcecc4">